fcl/detsim/detsim_2d_icarus.fcl

```diff
@@ -15,21 +16,39 @@ services: {
 physics: {
 
   producers: {
+    @table::icarus_shifting_triggersim.producers
+
     crtdaq:         @local::icarus_crtsim
      opdaq:         @local::icarus_simpmt
        daq:         @local::icarus_simwire_wirecell
 
        rns:         { module_type: "RandomNumberSaver" }
   } # producers
-  
-  simulate: [ rns, opdaq, daq, crtdaq ]
-  
+
+  simulate: [ rns, opdaq, @sequence::icarus_shifting_triggersim.path, daq, crtdaq ]
+
   # define the output stream, there could be more than one if using filters
   stream:  [ rootoutput ]
 
 } # physics
 
+physics.producers.crtdaq.G4ModuleLabel: "shifted"
 
 outputs: {
-  rootoutput: @local::icarus_rootoutput
+  rootoutput: {
+    @table::icarus_rootoutput
+    outputCommands: [
+      "keep *",
+      "drop *_ionization_*_*",
+      "drop *_pmtlvdsgatesinit_*_*",
+      "drop *_pmtfixedthrinit_*_*",
+      "drop *_pmttriggerwindowsinit_*_*",
+      "drop *_triggersimgatesinit_*_*"
+      ]
+  }
 }
+
+#legacy G4 configs
+services.Geometry.GDML: "icarus_complete_20220518_overburden.gdml"
+services.Geometry.ROOT: "icarus_complete_20220518_overburden.gdml"
+physics.producers.daq.wcls_main.configs: ["pgrapher/experiment/icarus/wcls-multitpc-sim-drift-simchannel-shifted.jsonnet"]
```

Comment:
This is confusing.
This PR should not need to affect the GEANT4 model... it sounds like the default has changed to the different model and here we manually revert to another one that is not default any more.
Is upstream `detsim_2d_icarus.fcl` broken? Can this be clarified? Also question for @cerati and @SFBayLaser, I suppose.

Reply:
To be clear about the purpose of this commit (3510f6ae8ce0cb7b48f92e43162c0034a4154d14), I had previously removed these lines, which would break the legacy LArG4 configurations. Here, I am restoring what's currently present on `develop` ([detsim_2d_icarus.fcl](https://github.com/SBNSoftware/icaruscode/blob/develop/fcl/detsim/detsim_2d_icarus.fcl)) with an override to point WireCell to the jsonnet where I have altered the input tags for the simulated energy deposits.

For the newer LArG4 workflow, I've modified  `detsim_2d_icarus_refactored.fcl` in the same commit I mention above.